timeseries/tests/test_Interface.py

```diff
@@ -0,0 +1,135 @@
+from Interface import SizedContainerTimeSeriesInterface
+from pytest import raises
+import reprlib
+import numpy as np
+import collections
+
+#define a concrete SizedContainerTimeSeriesInterface
+class concrete_SizedContainerTimeSeriesInterface(SizedContainerTimeSeriesInterface):
```

Comment:
class name should use CamelCase, see https://google.github.io/styleguide/pyguide.html